### PR TITLE
Add Cloud Native Aarhus meetups to meetups.yaml

### PR DESCRIPTION
**Pull request by kaspernissen**
Started working on adding Cloud Native Aarhus to the `meetups.yaml`

### CODE REVIEW

**Comment by kaspernissen** [comment]
@luxas quick question, how do we add links to relative paths for slides?

**Comment by kaspernissen** [comment]
@hoeghh @sazo @Steiniche do you mind pitching in with the rest of our meetups?

**Comment by kaspernissen** [comment]
That makes sense, didn't catch that at first. Thank you for clarifying.

**Comment by luxas** [comment]
Please don't indent all the other YAML to make this PR readable

**Comment by kaspernissen** [comment]
Sorry about that. Strange why it does that, haven't touched those sections other than using Visual Code to collapse parts of the file. Will fix this.

**Comment by kaspernissen** [comment]
Indention should be fixed now

File: meetups.yaml [hunk 1]
```diff
@@ -500,4 +539,61 @@ meetupGroups:
     - startTime: 20:30
       endTime: 21:00
       title: Feel free to hang out with the people for the rest of the evening, do some Q&A with the speakers, etc.
-    
\ No newline at end of file
+- name: Cloud Native Aarhus
+  meetupURL: https://www.meetup.com/Cloud-Native-Aarhus/
+  city: Aarhus, Denmark
+  organizers:
+  - Kasper Nissen ([@kaspernissen](https://github.com/kaspernissen)), CNCF Ambassador, Cloud Architect / SRE @lunarway, [kubecloud.io](https://kubecloud.io), kaspernissen@gmail.com
```
Comment (luxas) [inline meetups.yaml]: reference yourself from speakers

**Comment by luxas** [comment]
Can you make this non-draft?

**Comment by kaspernissen** [comment]
We are still missing 10 meetups - so it's still WIP

**Comment by kaspernissen** [comment]
All meetups has now been added to the `meetups.yaml`. We are missing support for multiple speakers (see issue #39) before completely done with this PR. Generator seems to be working after compiling the binary myself (see issue #40)

**Comment by luxas** [comment]
Now with #43 #42 #41 merged you should have an easier time finishing this PR @kaspernissen :wink:

**Comment by luxas** [comment]
The PR needs a rebase to pick up the YAML file changes I made at master

**Comment by kaspernissen** [comment]
@luxas great work! I have merged master and made the necessary changes. But, generation fails, when having multiple speakers on a presentation. If I remove the second speaker, the generation passes. Maybe you have an idea to where the problem is? 

Error message:
``` 
generator/bin/generator --dry-run=false
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x118d9a2]

goroutine 1 [running]:
main.(*Speaker).UnmarshalJSON(0xc0003ac200, 0xc0003a8218, 0x14, 0x3be8, 0x1659070, 0xc0003ac200)
        /Users/kaspernissen/Development/cloud-native-nordics/meetups/generator/types.go:99 +0x172
encoding/json.(*decodeState).literalStore(0xc0000b2868, 0xc0003a8218, 0x14, 0x3be8, 0x11bf520, 0xc0003693a8, 0x196, 0xc00012cb00, 0x10a6c82, 0x197)
        /usr/local/Cellar/go/1.11.4/libexec/src/encoding/json/decode.go:871 +0x2798
encoding/json.(*decodeState).value(0xc0000b2868, 0x11bf520, 0xc0003693a8, 0x196, 0x11bf520, 0xc0003693a8)
        /usr/local/Cellar/go/1.11.4/libexec/src/encoding/json/decode.go:397 +0x20e
....
```

**Comment by kaspernissen** [comment]
This is now ready for merge. It turns out it was a wrong reference breaking the build. I have added better error handling when this is the case. See PR #46

**Comment by kaspernissen** [comment]
I will merge this now.

**Pull request merged**